build/esbuild-build.ts

```diff
@@ -29,6 +29,7 @@ export const esBuildContext: esbuild.BuildOptions = {
     SUPABASE_STORAGE_KEY: generateSupabaseStorageKey(),
     commitHash: execSync(`git rev-parse --short HEAD`).toString().trim(),
     NODE_ENV: process.env.NODE_ENV || "development",
+    IS_RFC: JSON.parse(process.env.RFC!)
```

Comment:
It makes sense to provide a default value here in case `process.env.RFC` is empty, like `process.env.RFC || false`. This way we make it backwards compatible with already deployed version of `work.ubq.fi`.

Reply:
https://github.com/ubiquity/work.ubq.fi/pull/77/commits/f27d1d7c8ad76eff8ea968018b76e5c875e6fbf5